Tests/MongoSwiftTests/CommandMonitoringTests.swift

```diff
@@ -176,8 +175,13 @@ private struct CMTest {
             if let max = modifiers?["$maxTimeMS"] as? Int {
                 maxTime = Int64(max)
             }
+            var hint: Hint?
+            if let hintDoc = modifiers?["$hint"] as? Document {
```

Comment:
Historically, the [`$hint` modifier](https://docs.mongodb.com/manual/reference/operator/meta/hint/#metaOp._S_hint) has accepted either a string or document.

A more general question might be why we're supporting modifiers at all in Swift. Isn't the driver new enough that we should only need to concern ourself with the `find` command's API?

Reply:
there is only one case that uses modifiers and it stores the hint as a `doc`, so I didn't bother checking for a string, as I assume no more cases with modifiers will be added in the future.

we don't support providing a modifiers document, but our `FindOptions` contains all the fields one would have put in modifiers, so rather than skipping the test I parse them out to to the correct fields for the sake of more test coverage

Reply:
Oh, I didn't realize this was in a test file and thought the driver itself was actually supporting the deprecated "modifiers" option. Nevermind!